api/v1/harborcluster_types.go

```diff
@@ -164,7 +164,7 @@ type Oss struct {
 	Encrypt       string `json:"encrypt,omitempty"`
 	Secure        string `json:"secure,omitempty"`
 	Chunksize     string `json:"chunksize,omitempty"`
-	Rootdirectory string `json:"rootdirectory,omitempty"`
+	RootDirectory string `json:"rootdirectory,omitempty"`
```

Comment:
Json struct tag needn't update?

Reply:
yes, all params are lowercase, ref [here](https://github.com/goharbor/harbor-operator/blob/c8e922bded7080598ee5d10861271b792a22086d/docs/installation/registry-storage-configuration.md#create-the-secret)

Reply:
In my opinion, This struct of HarborCluster is the API of ours, We should follow the go naming convention. And the secret is the resource that we need to create, but not related to the HarborCluster field naming.

Reply:
https://docs.docker.com/registry/configuration/#storage all storage field are lowercase, I think we should follow this standard.